protocol/javatests/org/openyolo/protocol/RetrieveResultTest.java

```diff
@@ -58,15 +58,6 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void testInnerClassNewArray(){
```

Comment:
Took a closer look at RetrieveResult, it is a Parcelable but is missing it's implementation